src/Stratis.Bitcoin.Features.BlockStore/Models/BlockByHashQueryModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Stratis.Bitcoin.Features.BlockStore.Models
+{
+    public abstract class QueryModelBase
```

Comment:
For consistency, look at how we created and named those in https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs